debezium-connector-sqlserver/src/main/java/io/debezium/connector/sqlserver/SqlServerTaskConfig.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.debezium.connector.sqlserver;
+
+import org.apache.kafka.common.config.ConfigDef;
+
+import io.debezium.config.Configuration;
+import io.debezium.config.Field;
+
+public class SqlServerTaskConfig {
+    public static final Field TASK_DATABASE_NAMES = Field.create("task.database.dbnames")
```

Comment:
As we use `Field` i'd add at least validation logic it brings. See connector config for example:
```java
    public static Field.Set ALL_FIELDS = Field.setOf(CONFIG_DEFINITION.all());

    public static ConfigDef configDef() {
        return CONFIG_DEFINITION.configDef();
    }

    protected static final ConfigDefinition CONFIG_DEFINITION = ConfigDefinition.editor() ...
```
So we can execute the following right after object creation:
```java
config.validate(SqlServerTaskConfig.ALL_FIELDS)
```

Reply:
It means in SqlServerTaskConfig we need to store our own `config`. Which is a subset of `config` we get in constructor.

Reply:
That's unsafe from the API standpoint that an object that can be instantiated in a potentially invalid state needs to be additionally validated. There's no guarantee that whatever component instantiates it does the validation as well.

We need to think of a more robust API.